spec/text_normalizer/normalize_title_spec.rb

```diff
@@ -14,6 +14,10 @@
     it 'should reverse title parts when reverse is true' do
       expect(subject.normalize_title('', %w[title subtitle], '-', true)).to eq('subtitle-title')
     end
+
+    it 'should escape ampersand in title correctly' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.